pkg/kubelet/cm/internal_container_lifecycle_test.go

```diff
@@ -0,0 +1,121 @@
+package cm
+
+import (
+	"github.com/golang/mock/gomock"
+	"k8s.io/api/core/v1"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	kubefeatures "k8s.io/kubernetes/pkg/features"
+	"k8s.io/kubernetes/pkg/kubelet/cm/cpumanager"
+	"k8s.io/kubernetes/pkg/kubelet/cm/memorymanager"
+	"k8s.io/kubernetes/pkg/kubelet/cm/topologymanager"
+	"testing"
+)
+
+func TestPreStartContainer(t *testing.T) {
+	// setup vars
+
+	var tests = []struct {
+		i internalContainerLifecycleImpl
```

Comment:
please could you add a `name` field to cases to describe what they are trying to test?